content/soarian/r4/clinical/medications/immunization.md

```diff
@@ -0,0 +1,171 @@
+---
+title: Immunization | R4 API
+---
+
+# Immunization
+
+* TOC
+{:toc}
+
+## Overview
+
+The Immunization resource can include current and historical administration of vaccinations for a patient. The response can include administered immunizations and reported immunizations, including records reconciled from external sources such as the patient, state registries, and Consolidated Clinical Document Architecture (C-CDA) documents. The response does not include immunizations that may be ordered or scheduled, but have not been administered yet.
+
+Soarian Clinicals<sup>®</sup> supports a read-only Application Programming Interface (API). This API accepts `GET` and `POST` based [search] and `GET` based [read] interactions. The response represents the most current information about the patient that is charted in Soarian Clinicals<sup>®</sup> at the time of the query.
```

Comment:
Ok since this is new and I believe the POST based search may require a JSON Body(i could be wrong here), i would recommend giving out 1 example of POST based search. Also from the documentation it seems "GET" is more favoured for READ call and POST is for search.